libcxx/include/__chrono/formatter.h

```diff
@@ -517,6 +548,11 @@ _LIBCPP_HIDE_FROM_ABI constexpr bool __date_ok(const _Tp& __value) {
     return true;
   else if constexpr (same_as<_Tp, chrono::local_info>)
     return true;
+#  endif
+#  if !defined(_LIBCPP_HAS_NO_TIME_ZONE_DATABASE) && !defined(_LIBCPP_HAS_NO_FILESYSTEM) &&                            \
+      !defined(_LIBCPP_HAS_NO_LOCALIZATION)
+  else if constexpr (__is_specialization_v<_Tp, chrono::zoned_time>)
+    return true;
 #  endif
```

Comment:
We could perhaps still provide `zoned_time` when the TZDB / localization / filesystem is disabled. It's not a very useful type without those other features, but whatever.

The benefit would be that we don't need any complexity here because the type doesn't exist.